docs/reference/mapping/dynamic/field-mapping.asciidoc

```diff
@@ -1,35 +1,36 @@
 [[dynamic-field-mapping]]
 === Dynamic field mapping
 
-By default, when a previously unseen field is found in a document,
-Elasticsearch will add the new field to the type mapping.   This behaviour can
-be disabled, both at the document and at the <<object,`object`>>  level, by
-setting the <<dynamic,`dynamic`>> parameter to `false` (to ignore new fields) or to `strict` (to throw
-an exception if an unknown field is encountered).
+When {es} detects a new field in a document, it _dynamically_ adds the field to
+the type mapping. The <<dynamic,`dynamic`>> parameter controls this behavior.
 
-Assuming `dynamic` field mapping is enabled, some simple rules are used to
-determine which data type the field should have:
+You can disable dynamic mapping, both at the document and at the
+<<object,`object`>> level. Setting the `dynamic` parameter to
+`false` ignores new fields, and `strict` throws an exception if {es} encounters
```

Comment:
What d you think about s/throws an exception/rejects the document/ ?  "throws an exception" is a very java-implementation level thing. The result of it is to reject the document.

Reply:
Definitely more clear. I'll make that substitution.